### ISSUE TITLE
Package app as a single portable executable

### ISSUE THREAD

**Issue by teoxoy** [eksime/VDesk#60]
It would be great if I could use the app without installing it.
Can you maybe package the app as a single portable executable (i.e. VDesk.exe)?

**Comment by starik1968** [comment]
It is an MSI-file, it can be unpacked and then the .exe used in my opinion

**Comment by teoxoy** [comment]
MSI is an installer. And even if you install it and then try to use VDesk.exe by itself it doesn't work; it has dependencies in the folder where it got installed.

**Comment by eksime** [comment]
I could certainly upload the exe and dlls in a .zip or
(licences permitting), I could look at building an exe with the DLLs built in (like a c# version of a statically linked binary)

The msi is mostly responsible for adding the install folder to your PATH and updates, it's pretty basic otherwise.

In the meantime, I believe you could extract the DLLs and exe from the msi using msiexec's `/a` flag

**Comment by teoxoy** [comment]
> I could look at building an exe with the DLLs built in (like a c# version of a statically linked binary)

This is what I meant. An exe that you can just carry around.
It would be great if you can release it like this.

**Comment by starik1968** [comment]
It is an MSI-file, it can be unpacked and then the .exe used in my opinion

**Comment by starik1968** [comment]
I unpacked it and all run well as far as I remember

**Comment by teoxoy** [comment]
Yes it does but it has dependencies in the folder where it got installed so you can't just move the exe around and use it. You need all the files which could be packaged in the exe. - This was my suggestion.
Edit: Even if you don't use the msi to install it and just extract the files, you still need to carry around the dependencies with the exe.

**Comment by EED85** [comment]
Hi,
would be great, it a portable app would be working easily ...
Eric
